### PR TITLE
API

### PR DESCRIPTION
**Pull request by anovis**
Add API tab.

### CODE REVIEW

**Comment by bensonlee5** [comment]
1. generally - 
* capitalize Flow as defined term
* capitalize Agent as defined term
* use "a" if subsequent word starts with consonant or sounds like it starts with a consonant; an if vowel (getConnection, getFlow, getFlowRuns, getFlowRuns)

examples:
getFlow - change an to "a"
getFlowRun - update description to "Get a Flow run in an environment by its id"
getFlowRuns - similarly, "Get all Flow runs for a specific Flow in an environment"

2. is there a blurb that could be added on Ganymede API (or perhaps some demonstration)?  I know it's probably obvious to users, but may make the page more welcoming

3. No need to have Ganymede API as top-level category (just remove that 1 level of nesting)

**Review by bensonlee5 — APPROVED** [review]

**Comment by bensonlee5** [comment]
one other comment - it could make sense to add hyperlinks to the page where the different items are discussed, in case people navigate first to the API and then get to the App section from there

File: docs/api/download-file.api.mdx [hunk 1]
```diff
@@ -0,0 +1,154 @@
+---
+id: download-file
+title: "downloadFile"
+description: "downloadFile"
+sidebar_label: "downloadFile"
+hide_title: true
+hide_table_of_contents: true
+api: eJy9U0uP2yAQ/itoTq2E1kmP3PbQRqteqj5OUVQRexLTxZiFcdoI8d+rwY9mk90eeujJmBmG70WC3mPQZHr30ICCpv/pbK+bD8YiSPA66A4JQwS1TeB0h6AA3cmE3nXoCCQEfBpMwAYUhQElGAcKvKYWJMS6xU6DSkBnz0cjBeOOkLNcppXPv47Z8cHoexcxcv3dasWfBmMdjGdaoGCDJLSI5uiwEUOwgnoxExVaHIxFYZzQTlwwE/uzMBTFBK/uHTFflUB7b01dNKt+RL4h/QVhzllCxHoIhs5FRe3N90fk9Y7xkz6yusCSR9jJK/BXjnRIbc9OHZGKQdSCguq0ri6wV+niJ1dMMFaJmeRqnseyYjjN1g7BgoKWyEdVVantI2W4xgISTjoYvbej2tw1yn3QgyUewDu5+DJz/sLSjP0L80Um7c1HPMNkd4u6wQByjsbczxIad+jLQUOWaxvtzh02KO4/PdwAvSoyy7GwvlvdrSY7dc3gebbvI3W6GDndvMRjkv3Z+AU94S+qvNXGQZajhGmyZAunNcirp6Ke/xZfeHvK2GLNTk7SbiGlvY74LdiceftpwFCS88eJYl9jIq8bUAdtI94AXuILbz5PD+2tAPkykdkbdy6G2wFH74t1zyjxM/6PVxed8i7LOSlMfSzd1zV6ujh080o5k8vr2bz/ChL0wFZdhvGxhHFa8PQXAc25nHGVjMpXmlMaQ57z0j+WXj2xPIOxm5XZ5Zx/A4Xs7C0=
```
Comment (mattcoganymede) [inline docs/api/download-file.api.mdx:8]: Are these keys intended to be committed -- these seem auto-generated, so maybe expected.

Comment (anovis) [reply docs/api/download-file.api.mdx]: @mattcoganymede yeah they are autogenerated. some sort of uuid or hash it seems

**Comment by bensonlee5** [comment]
- In the API intro, mention how users can find ids for Agents, Flows, etc. (add pointer to [System Reference Tables](https://website-docusaurus-git-openapi-docs-ganymede.vercel.app/app/flows/SystemMetadata))
- maybe subsequent to initial release, but would be good to know rate limits

- for postQuery, add a link or comment that SQL has to be compatible with BigQuery
- for getFile / getFiles, add a pointer to the [get_gcs_uri method](https://website-docusaurus-git-openapi-docs-ganymede.vercel.app/sdk/FileTags#get_gcs_uri)
- getFlowRun has misspelled param name for environmentVersion

**Comment by bensonlee5** [comment]
> > * In the API intro, mention how users can find ids for Agents, Flows, etc. (add pointer to [System Reference Tables](https://website-docusaurus-git-openapi-docs-ganymede.vercel.app/app/flows/SystemMetadata))
> > * maybe subsequent to initial release, but would be good to know rate limits
> > * for postQuery, add a link or comment that SQL has to be compatible with BigQuery
> > * for getFile / getFiles, add a pointer to the [get_gcs_uri method](https://website-docusaurus-git-openapi-docs-ganymede.vercel.app/sdk/FileTags#get_gcs_uri)
> > * getFlowRun has misspelled param name for environmentVersion
> 
> added pr in api server addressing feedback. will add a ticket to add rate limiting specific to the public api and then will include in the documents.

cool - for reference, [this Benchling rate limits section](https://docs.benchling.com/docs/rate-limiting#rate-limits) might be a good reference since we could get comp'd to this.  I can flesh out details (e.g. - add that disclaimer at the top like they did) once the bones are in

**Review by bensonlee5 — APPROVED** [review]

**Comment by bensonlee5** [comment]
- be consistent about adding period to end of description sentence (add everywhere? - e.g. - missing on getConnection description)
- postQuery - Bigquery is misspelled as Biqquery
- FlowRunState and FlowRun - should "unknown" be titlecased as "Unknown"?
- TableRow - property name has a * at the end (not sure if this was intentional)

**Pull request merged**